animation_nodes/data_structures/__init__.py

```diff
@@ -3,6 +3,9 @@ def importDataStructures():
     from . struct import ANStruct
 
     from . meshes.mesh_data import Mesh
+    from . gpencils.gpencil_data import Layer
+    from . gpencils.gpencil_data import Stroke
+    from . gpencils.gpencil_data import Frame
```

Comment:
I think the names should be prefixed with `GP`, as the names are a bit ambiguous right now.

Reply:
Okay. Should I also change the name of the socket files?

Reply:
They are in their own directory, so that won't be necessary.

Reply:
Okay.